jobs/update-stale-payment/config.py

```diff
@@ -105,6 +105,13 @@ class _Config(object):  # pylint: disable=too-few-public-methods
     # REPORT API Settings
     REPORT_API_BASE_URL = os.getenv('REPORT_API_BASE_URL')
 
+        # NATS Config
```

Comment:
You need to put these values into openshift template for tracking environment purpose.

Reply:
Done